cEP-0024.md

```diff
@@ -0,0 +1,248 @@
+# Implement GitMate automations and plugins for coala
+
+| Metadata |                                                       |
+| -------- | ----------------------------------------------------- |
+| cEP      | 24                                                    |
+| Version  | 1.0                                                   |
+| Title    | Implement Gitmate automations and plugins for coala   |
+| Authors  | Vamshi Krishna Bommerla <mailto:bommvams72@gmail.com> |
+| Status   | Proposed                                              |
+| Type     | Process                                               |
+
+## Abstract
+
+This cEP describes the improvements to the coala workflow and the necessary
+automation in [GitMate](https://gitmate.io/) to realise it, as a part of the
+[GSoC project](https://summerofcode.withgoogle.com/projects/#5344316227059712).
+
+## Introduction
+
+[GitMate](https://gitmate.io/) is an automation tool for developers. While it
+works well, there are a number of missing features which would save time of
+coala community, for example automatically close PRs opened on master of forks.
+Many of the plugins in GitMate are general plugins, but coala needs more
+plugins which support coala development workflow. This project adds plugins to
+auto-review, auto-reject, auto-label PRs, assign, unassign issues, squash
+commits.
+
+## Changes in coala workflow
+
+### Use increasing difficulty level order to assign issues
+
+Current assign process in coala is to check team membership of a user,
+difficulty level of the issue and assign issue to user if eligible. To
+implement current assign process in GitMate, IGitt would need to support teams,
+which is difficult to implement, because there are no teams in GitLab.
+
+Considering groups as teams in GitLab is a possible solution for IGitt
```

Comment:
"groups" - where did that come from?

You need to explain that GitLab has groups instead of teams, and explain that they are different.

Include links to the GitHub teams and GitLab groups documentation.